### PR TITLE
Release v0.28.0

### PR DESCRIPTION
**Pull request by hiddeco**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+**Release date:** 2022-12-20
+
+This prerelease disables the caching of Secret and ConfigMap resources to
+improve memory usage. To opt-out from this behaviour, start the controller
+with: `--feature-gates=CacheSecretsAndConfigMaps=true`.
+
+In addition, a new flag `--graceful-shutdown-timeout` has been added to
+control the duration of the graceful shutdown period. The default value is
+`-1` (disabled), to help prevent releases from being stuck due the controller
+being terminated before the Helm action has completed.
```
Comment (relu) [inline CHANGELOG.md:14]:
```suggestion
In addition, a new flag `--graceful-shutdown-timeout` has been added to
control the duration of the graceful shutdown period. The default value is
`-1` (disabled), to help prevent releases from being stuck due to the controller
being terminated before the Helm action has completed.
```

**Review by relu — CHANGES_REQUESTED** [review]
Just a nit, otherwise all good.

**Review by relu — APPROVED** [review]
LGTM :+1:

**Review by stefanprodan — APPROVED** [review]

**Comment by stefanprodan** [comment]
@relu can you please update https://github.com/fluxcd/website/pull/1316 and include the feature flag?

**Comment by stefanprodan** [comment]
@relu here is an example of how to display the feature gates https://github.com/fluxcd/website/blob/main/content/en/flux/components/source/options.md#feature-gates

**Pull request merged**